data/hai/hai culture conversations.txt

```diff
@@ -329,3 +329,56 @@ mission "Hai Film Release"
+			choice
+				`	(Enter the restaurant.)`
+					goto restaurant
+				`	(Buy something from the human food vendor instead.)`
+				`	(On second thought, I'm not that hungry.)`
+					decline
+			action
+				payment -25
+			`	Food made by humans is certainly the safer option. The vendor sells you a dish comprised of steamed vegetables, along with a fillet of some yellow-tinged fish which she assures you is perfectly edible. The meal is somewhat bland, but the fish is buttery and tender. You dispose of the leftovers in a nearby garbage receptacle, and you watch as it beeps melodically, incinerating the refuse.`
+				decline
+			label restaurant
+			`	A Hai restaurant is sure to provide a more unique experience. The door jingles a short melody as you enter the establishment, and a middle-aged female Hai with her back to you shouts something in the Hai language.`
+			branch translator
+				has "language: Hai"
+			`	You freeze for a second, as it is unclear what the Hai wants from you.`
+				goto order
+			label translator
+			`	Your translator chirps dutifully, "Have a [seat, rest] anywhere you like! I'll be right with you."`
+			label order
+			`	Only a short moment later, the same Hai hostess peers over at you and chitters in what you assume is embarrassment. "I'm so sorry. I didn't realize I would have the pleasure of serving a human today. Please, have a seat anywhere at the bar. I'll come to take your order shortly."`
+			`	You take a look around and note that the restaurant is actually quite crowded. There are Hai dressed in fancy tuxedos, fashionable clothing, casual dress, and even some wearing peculiar overalls with holes in the backs for their tails; there don't seem to be any other humans. With only a few seats available, you select one on the end, next to a rather large Hai wearing green overalls and an orange straw hat. He grunts noncommittally as you sit down on a stool clearly designed for an organism with a tail. Before long, the hostess approaches and asks for your order.`
+			choice
+				`	"I'll have today's special."`
+				`	"I'd like some soup, please."`
+			`	"You're in luck," she says. "Today only, each meal is served with a free side of acorn bread." She scampers off to place and retrieve your order.`
+			`	The Hai next to you grunts again - this time with clear frustration - and turns to speak. "Acorn bread! That's all they serve here anymore. Ever since you humans arrived in our space, that's all we get to eat. Traditional Hai breadmaking has become a lost art."`
+			choice
+				`	"You mean there's more than just acorn bread?"`
```

Comment:
https://github.com/endless-sky/endless-sky/pull/9738#discussion_r1520371683
This choice is somewhat incongruous with the rest of the conversation as they currently are.

Reply:
How about this:
```suggestion
				`	"Traditional Hai breadmaking?"`
```